FinnUI.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = 'FinnUI'
-  s.version      = '22.3.0'
+  s.version      = '23.0.0'
   s.summary      = "FINN's iOS UI Features"
   s.author       = 'FINN.no'
   s.homepage     = 'https://schibsted.frontify.com/d/oCLrx0cypXJM/design-system'
```

Comment:
We should update the podspec to iOS 13 as well 😄

Reply:
Nice catch 😉